### ISSUE TITLE
Getting KeyError: 'conversationId'

### ISSUE THREAD

**Issue by anzarwani** [Soulter/hugging-chat-api#30]
I exported cookies from an incognito session and pasted the same in my cookies.json file. After that I ran the sample code but I am getting this error. 

Could extracting cookies from an incognito session be the reason for this error?

**Comment by LyubimovVladislav** [comment]
The cookies should be as list of objects and should include `name` and `value` properties. 
Example of minimal working cookie:
```
[
  {
  "name": "hf-chat",
   "value": "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
  }
]
```

**Comment by anzarwani** [comment]
@LyubimovVladislav It is already a valid cookie.

**Comment by anzarwani** [comment]
I realized I had turned on cookie and popup blocker extension in my web browser because of which the number of lines in my cookies.json where less. However, after turning them off, and after getting valid cookies, I still return with the same error. I tried capturing cookies from different web browsers but the error remains the same.

I was wondering if I am missing any important step? I am not trying to run any complex code. I am just following the template example.
